listenbrainz/webserver/templates/navbar.html

```diff
@@ -24,30 +24,30 @@
           {% endif %}
         {%- endif -%}
         <li><a href="{{ url_for('index.recent_listens') }}">Recent</a></li>
-        <li><a href="{{ url_for('profile.import_data') }}">Import</a></li>
         <li class="dropdown">
           <a class="dropdown-toggle" data-toggle="dropdown" href="#">
              About<span class="caret"></span>
           </a>
           <ul class="dropdown-menu" role="menu">
-              <li><a href="{{ url_for('index.faq') }}">FAQ</a></li>
-              <li><a href="{{ url_for('index.goals') }}">Goals</a></li>
-              <li><a href="{{ url_for('index.current_status') }}">Current status</a></li>
-              <li><a href="https://metabrainz.org/gdpr">GDPR compliance</a></li>
-              <li><a href="https://metabrainz.org/privacy">Privacy policy</a></li>
-              <li><a href="https://metabrainz.org">About MetaBrainz</a></li>
+            <li><a href="{{ url_for('index.faq') }}">FAQ</a></li>
+            <li><a href="{{ url_for('index.goals') }}">Goals</a></li>
+            <li><a href="{{ url_for('index.roadmap') }}">Roadmap</a></li>
+            <li><a href="https://listenbrainz.readthedocs.io">API Docs</a></li>
+            <li><a href="{{ url_for('index.contribute') }}">Contribute</a></li>
+            <li><a href="{{ url_for('index.current_status') }}">Current site status</a></li>
+            <li><a href="https://metabrainz.org/gdpr">GDPR compliance</a></li>
+            <li><a href="https://metabrainz.org/privacy">Privacy policy</a></li>
+            <li><a href="https://metabrainz.org">About MetaBrainz</a></li>
           </ul>
         </li>
-
         <li class="dropdown">
           <a class="dropdown-toggle" data-toggle="dropdown" href="#">
-             Contribute<span class="caret"></span>
+             Data<span class="caret"></span>
           </a>
           <ul class="dropdown-menu" role="menu">
-            <li><a href="{{ url_for('index.contribute') }}">Contribute</a></li>
-            <li><a href="{{ url_for('index.data') }}">Data</a></li>
-            <li><a href="{{ url_for('index.api_docs') }}">API Docs</a></li>
-            <li><a href="{{ url_for('index.roadmap') }}">Roadmap</a></li>
+            <li><a href="{{ url_for('index.add_data_info') }}">Adding data</a></li>
```

Comment:
I think the adding data page should be in the top level instead of a dropdown, maybe with a better title? "Track your listens" or something that's more easily understandable for users?

Reply:
@paramsingh Apologies on disappearing on this one, I had some family issues come up. I'll get this sorted some time in the next day or two!